docs/user_guide/branding.rst

```diff
@@ -77,41 +77,43 @@ To reference an external website, make sure your link starts with ``http``:
        }
    }
 
-Customize logo alternative text
--------------------------------
-
-You may set a custom ``alt text`` for your logo to replace the default ``"logo image"`` generic description.
-Adding a descriptive ``alt text`` can help make your documentation more accessible to readers using screen readers or another assistive tech.
+Add a logo title
+----------------
 
-To do so, customize the ``html_theme_options["logo"]["alt_text"]`` configuration option as in the following example:
+To add a title in the brand section of your documentation, define a value for ``html_theme_options.logo["text"]``.
+This title will appear next to the logo image if set.
 
 .. code-block:: python
-   :caption: conf.py
 
    html_theme_options = {
        "logo": {
-           # Because the logo is also a homepage link, including "home" in the alt text is good practice
-           "alt_text": "My Project Name - Home",
+           "text": "My awesome documentation",
        }
    }
 
-Add a logo title
-----------------
+.. note:: The ``html_title`` field will work as well if no logo images are specified.
 
-To add a title in the brand section of your documentation, define a value for ``html_theme_options.logo["text"]``.
-This title will appear next to the logo image if set.
+Customize logo alternative text
```

Comment:
I moved this section below the logo title because I'm thinking that with the new default behavior the theme consumer may have no reason to change the default alt text value.